azure-pipelines.yml

```diff
@@ -278,8 +279,9 @@ stages:
               env:
                 TWINE_USERNAME: "qiskit"
                 TWINE_PASSWORD: $(TWINE_PASSWORD)
-        - job: 'qiskit-pkg'
+        - job: 'qiskit_pkg'
           pool: {vmImage: 'ubuntu-latest'}
+          dependsOn: "Deploy"
```

Comment:
I think that I need to move this up to the stage level not at the job level.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/10663/commits/7b6668512136dda5f75e38f40601e6545c101d3b